src/node_file.cc

```diff
@@ -479,9 +479,9 @@ void FillStatsArray(double* fields, const uv_stat_t* s) {
   fields[9] = -1;
 #endif
   // Dates.
-#define X(idx, name)                          \
-  fields[idx] = (s->st_##name.tv_sec * 1e3) + \
-                (s->st_##name.tv_nsec / 1e6); \
+#define X(idx, name)                                                 \
+  fields[idx] = (static_cast<uint32_t>(s->st_##name.tv_sec) * 1e3) + \
+                (static_cast<uint32_t>(s->st_##name.tv_nsec) / 1e6); \
```

Comment:
so that's what we did in #12607 we moved the type coarctation to the left side with `1e3`. Now with my patch it `uint32_t / double` so it's 64bit floating point arithmetic.